lib/slack_transformer/html/unsupported.rb

```diff
@@ -0,0 +1,16 @@
+module SlackTransformer
+  class Html
+    class Unsupported
+      attr_reader :input
+
+      def initialize(input)
+        @input = input
+      end
+
+      def to_slack
+        # Filter out all remaining < > tags except for Slack links (<http://url|text> or <http://url>)
+        input.gsub(/<(.[^http][^|]*?)>/, '')
+      end
```

Comment:
I wrote this transformer to clean up remaining HTML tags for now with the plan of replacing it with some sort of "cleaner" class that does this and eliminates the spacing bug. As mentioned [here](https://vendr.atlassian.net/jira/software/projects/TS/boards/17?selectedIssue=TS-337).